### ISSUE TITLE
Cannot read property 'setRawMode'

### ISSUE THREAD

**Issue by numanWD** [marrone/grunt-tinypng#14]
Hi
I am getting an error only with the pngs.

The message error is:

Running "tinypng:compress_png" (tinypng) task
Warning: Cannot read property 'setRawMode' of null Use --force to continue.

Thanks

**Comment by marrone** [comment]
I've not ran into this issue before. Can you supply one of the images you this error occurs with?

**Comment by gabrielmaldi** [comment]
I can confirm this, it happens with all my images. The problem seems to be Node.js 4, when I reverted to 0.10 (using nvm) everything works fine again.

I'll leave one image which causes the error here, but I think it's related to Node.js.

Thanks!

![example](https://cloud.githubusercontent.com/assets/3728897/10301808/cca4ff5c-6be4-11e5-8284-aa2f4654a473.png)

**Comment by danparm** [comment]
Happening to me as well. Although it doesn't appear to be a problem with an image or images. I have two tasks, the first one runs fine, then when the second runs I get the 'setRawMode' warning/error. If I comment the first task in my Gruntfile the second runs fine. The order of the tasks doesn't appear to matter, always chokes on the second one.

**Comment by HugoHeneault** [comment]
I have this issue too. :'(

**Comment by txcbenji** [comment]
Just ran in to this same problem today. :-(

**Comment by fonziemedia** [comment]
Hi, I also have this issue and also on the 2nd task, has anyone managed to get around it without reverting to an older version? Cheers

**Comment by marrone** [comment]
I just ran into this for the first time on a jpg when testing. It only happens for me when using **showProgress:true** setting

For everyone else who has encountered the issue, are you also using **showProgress**? Can you verify that if you turn the progress display off that you dont get the error?

**Comment by gabrielmaldi** [comment]
@marrone Confirmed. I just got the error with `showProgress: true`, then I changed it to `false` and everything worked fine.

**Comment by HugoHeneault** [comment]
@marrone Without `showProgress: true`, it's working. 
But it was cool to look at some progress while working with lots of images... :sob:

**Comment by HugoHeneault** [comment]
This happens when using grunt-tinypng with grunt watch, if it can help.

**Comment by marrone** [comment]
Thanks, I'll debug it further when I have a chance, at least now I know how to repro it.

**Comment by gabrielmaldi** [comment]
Hey, @marrone, have you had a chance to look into this?

I just tried v0.6.1 on node v8.11.1 and the issue still repros.

Perhaps it's caused by some outdated package dependency?

Thanks!

**Comment by HugoHeneault** [comment]
@gabrielmaldi Try to disable `showProgress` option :)

**Comment by gabrielmaldi** [comment]
@HugoHeneault yes, I did that [two years ago](https://github.com/marrone/grunt-tinypng/issues/14#issuecomment-243009809); I just wanted to know if we could get the progress indicator back 😄. Thanks!

**Comment by HugoHeneault** [comment]
Woops I didn't read right. My bad 😂

**Comment by sbwrege2z** [comment]
I can confirm the same behavior described here.
